Tasks/Gorbachev/Task3_1.py

```diff
@@ -0,0 +1,44 @@
+'''1. Написать программу для поиска наиболее часто встречающихся элементов в строке.
+Пользователь вводит строку, и количество элементов, которые он хотел бы увидеть в ответе.
+
+Пример:
+Строка:lkseropewdssafsdfafkpwe
+Элементы:3
+'s' встречается 4 раза
+'e' встречается 3 раза
+'f' встречается 3 раза'''
+from collections import OrderedDict
+
+def symbols(n, s):  
+    value_list = {}
+    for i in range(0, len(s), 1):
+            if s[i] in value_list:
+                value_list[s[i]] += 1
+            else:
+                value_list[s[i]] = 1
+    value_list = (OrderedDict(sorted(value_list.items(), key=lambda t: t[1], reverse=True)))
+
+    if n > len(value_list): # if lenght of symbols that should be shown are more than lenght of string
+        n = len(value_list)
+    symbol_count = list(value_list)[0:n] 
+
+    for key in symbol_count:
+        print(f'Symbol "{key}" used {value_list[key]} times.')
+    return()
```

Comment:
a lot of actions) make sure that you understand what lambda does, and do not use return with a parenthesis, do not use it at all if you do not want to return anything